shared/src/main/scala/mouse/any.scala

```diff
@@ -0,0 +1,13 @@
+package mouse
+
+
+trait AnySyntax {
+  implicit final def optionSyntaxMouse[A](oa: A): AnyOps[A] = new AnyOps(oa)
+}
+
+final class AnyOps[A](val oa: A) extends AnyVal {
+  def |>[B] (f: A => B) = f(oa)
+  def thrush[B] (f: A => B) = f(oa)
+
+  def ignore = ()
```

Comment:
`def ignore: Any => Unit = _ => ()` on the package object rather than as an enrichment?

That would let you write `"anythingAtAll" |> ignore shouldBe ()`